src/debug.h

```diff
@@ -53,6 +53,12 @@
 #define __FUNCTION_NAME__ __func__
 #endif
 
+#define fatal(cond) \
+    if( !( cond ) ) { \
+        debugmsg( "Assertion failed: %s", STRING( cond ) ); \
+        std::terminate(); \
```

Comment:
No, we have enough non-standard idioms. Anyone external to the project is going to expect `assert` to have certain characteristics.

We need to fix that `debugmsg` problem anyway.

I'm going to rename `fatal` to `enforce` - how does that sound?